resources/views/modals/edit_profile_modal.blade.php

```diff
@@ -188,6 +188,18 @@
                     </div>
                     @endif
 
+                    @if(Auth::user()->level() >= $constants['access_level']['admin'])
+                    <div class="form-group">
+                      <label>Antelope Password</label>
+                      <button type="button" class="form-control btn btn-outline-warning btn-fw" onclick="changePasswordPopup()">Change User Password</button>
+                    </div>
+                    @else
```

Comment:
Remove everything below the else handle (remove line 196-200)
There really isn't a point for there to be an Antelope username field hidden below the configuration above.
If you inspect element, there's two of them hidden twice now:
![image](https://user-images.githubusercontent.com/33847385/73538692-eb4a0280-442b-11ea-8245-59cad92fb055.png)

Reply:
Yeah I didn't mean to leave that in, good spot.